package.json

```diff
@@ -4,9 +4,6 @@
   "version": "4.236.1",
   "main": "./website/server/index.js",
   "dependencies": {
-    "@babel/core": "^7.18.6",
-    "@babel/preset-env": "^7.18.6",
-    "@babel/register": "^7.18.6",
```

Comment:
I think these package and package-lock changes are breaking the build. What's this removal for?

Reply:
I have no idea how those got changed--I didn't even open those files. I'll revert both to develop.